### ISSUE TITLE
'nrn.Segment' object has no attribute 'hname'

### ISSUE THREAD

**Issue by sunzhenyang2018** [orena1/neuron_reduce#14]
As mentioned in title, I encountered this error while running subtree_reductor

![reduce_issue1](https://user-images.githubusercontent.com/65326026/89326615-6d1b2280-d63f-11ea-8775-45b94d2341d9.JPG)

**Comment by orena1** [comment]
Strange, parent() should return a section and not a segment - 
neuron.yale.edu/neuron/static/py_doc/modelspec/programmatic/topology/secref.html#SectionRef.parent
what version of NEURON do you use?

**Comment by sunzhenyang2018** [comment]
Neuron 7.7.2
I noticed from the link you provided that the syntax is SecRef.parent, without the parenthesis. Might this be an issue?

**Comment by orena1** [comment]
You are correct, this is probably because your axon is a parent of the soma. Strange, either I expected this use-case but not tested it, or that something in NEURON changed.
Anyway, if you can send me a self-contained version of your model I can fix it.

Thanks

**Comment by sunzhenyang2018** [comment]
Sure. The hoc file I have is really long, because the majority of it is topol(). I can also send you a swc file of the morphology if you want.
[init_template.zip](https://github.com/orena1/neuron_reduce/files/5029170/init_template.zip)

**Comment by orena1** [comment]
Thanks @sunzhenyang2018  but I also need the modfiles
